.github/CODEOWNERS

```diff
@@ -1,7 +1,10 @@
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# these owners will be requested for review when someone
-# opens a pull request.
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
 * @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
+
+# These folks own any files in the /.github directory at the root of
```

Comment:
The `/` on this line is a bit redundant since you go on to mention it's at the root of the repo.  Do with this information what you will.

Reply:
I vote for removing the leading `/` in the comment because it has different connotations in a file system. The GitHub syntax examples also eschew it in the comment description.

Reply:
See commit 5b199bc.